### PR TITLE
:broom: Update README

### PR DESCRIPTION
**Pull request by misterpantz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,6 +234,7 @@ If you're interested in writing your own policies or contributing policies back
 | Microsoft 365 accounts           | `ms365`                    | `cnspec scan ms365 --tenant-id TENANT_ID --client-id CLIENT_ID --certificate-path PFX_FILE`                                                           |
 | Microsoft Azure subscriptions    | `azure`                    | `cnspec scan azure --subscription SUBSCRIPTION_ID`                                                                                                    |
 | Microsoft Azure virtual machines | `ssh`                      | `cnspec scan ssh user@host`                                                                                                                           |
+| Oracle Cloud Infrastructure (OCI) | `oci`                     | `cnspec shell oci`                                                                                              |
```
Comment (tas50) [inline README.md]:
```suggestion
| Oracle Cloud Infrastructure (OCI) | `oci`                     | `cnspec scan oci`                                                                                              |
```

File: README.md [hunk 2]
```diff
@@ -198,50 +200,53 @@ cnspec bundle upload mypolicy.mql.yaml
...
+| Google Workspace                  | `google-workspace`         | `cnspec shell google-workspace --customer-id CUSTOMER_ID --impersonated-user-email EMAIL --credentials-path JSON_FILE`                            |
+| Kubernetes cluster nodes          | `local`, `ssh`             | `cnspec scan ssh user@host`                                                                                                                       |
+| Kubernetes clusters               | `k8s`                      | `cnspec scan k8s`                                                                                                                                 |
+| Kubernetes manifests              | `k8s`                      | `cnspec scan k8s manifest.yaml `                                                                                                                  |
+| Kubernetes workloads              | `k8s`                      | `cnspec scan k8s --discover pods,deployments`                                                                                                     |
```
Comment (tas50) [inline README.md]: Let's nuke this line. Clusters is actually clusters + workloads

File: README.md [hunk 3]
```diff
@@ -198,50 +200,53 @@ cnspec bundle upload mypolicy.mql.yaml
...
+| GitLab groups                     | `gitlab`                   | `cnspec scan gitlab --group mondoohq`                                                                                                             |
+| Google Cloud projects             | `gcp`                      | `cnspec scan gcp`                                                                                                                                 |
+| Google Workspace                  | `google-workspace`         | `cnspec shell google-workspace --customer-id CUSTOMER_ID --impersonated-user-email EMAIL --credentials-path JSON_FILE`                            |
+| Kubernetes cluster nodes          | `local`, `ssh`             | `cnspec scan ssh user@host`                                                                                                                       |
+| Kubernetes clusters               | `k8s`                      | `cnspec scan k8s`                                                                                                                                 |
```
Comment (tas50) [inline README.md]:
```suggestion
| Kubernetes clusters & workloads              | `k8s`                      | `cnspec scan k8s`                                                                                                                                 |
```

File: README.md (hunk 2 above)
Comment (chris-rock) [inline README.md]:
```suggestion
```

**Review by chris-rock — APPROVED** [review]
Thank you @misterpantz and @tas50

**Pull request merged**